Backport of https://github.com/xcp-ng/branding-xcp-ng/pull/2 for 8.2
Reformat EULA file with shorter lines for xsconsole display.

diff --git c/branding/EULA w/branding/EULA
index 6e5aba4..561f26a 100644
--- c/branding/EULA
+++ w/branding/EULA
@@ -1,7 +1,8 @@
-The programs included with the XCP-ng system are free software. To know more about licenses of all programs, please read each COPYING file in the /usr/share/doc/* folders.
+The programs included with the XCP-ng system are free software.
+To know more about the licenses of all programs,
+please read each COPYING file in the /usr/share/doc/* folders.

 XCP-ng comes with no warranty, to the extent permitted by applicable law.

 For more info on XCP-ng, please check https://xcp-ng.org
-Professional support for production environment is available on https://xcp-ng.com
-
+Pro support for production environment is available on https://xcp-ng.com
